testing/model-mesh/7-analyze.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o errtrace
+set -x
+
+results_dir=$ARTIFACT_DIR/results
+oc get nodes -oyaml > $results_dir/artifacts-sutest/nodes.yaml
+oc get pods -A -owide | grep modelmesh > $results_dir/artifacts-sutest/pods.yaml
+oc get inferenceservices -A > $results_dir/artifacts-sutest/inferenceservices.yaml
+oc get runtimeserver -A > $results_dir/runtimeserver.status
```

Comment:
wrong name `runtimeserver`, I'm not sure what's the correct one :/

Reply:
Not sure either, keeping that one open

Reply:
it's `servingruntimes` you want here (I saw it above in the PR)